taxcalc/parameters.py

```diff
@@ -33,24 +37,36 @@ class CompatibleDataSchema(ma.Schema):
 
 class Parameters(pt.Parameters):
     """
-    Base Parameters class that wraps ParamTools, providing parameter indexing
-    for tax policy in the adjust method and backwards-compatible preserving
-    layer that supports Tax-Calculator's conventional reform formatting style
-    as well as convenience methods like set_Year for classes operating on this
-    one.
+    Base class that wraps ParamTools, providing parameter indexing
+    for tax policy in the ``adjust`` method and convenience methods
+    like ``set_year`` for classes inheriting from it. It also provides
+    a backwards-compatible layer for Tax-Calculator versions prior to 3.0.
 
     The defaults file path may be set through the defaults class attribute
-    variable or through the old DEFAULTS_FILE_NAME/DEFAULTS_FILE_PATH work
-    flow.
+    variable or through the ``DEFAULTS_FILE_NAME`` /
+    ``DEFAULTS_FILE_PATH work`` flow.
 
     A custom getter method is implemented so that the value of a parameter
     over all allowed years can conveniently be retrieved by adding an
-    underscore before the variable name (e.g. EITC_c vs _EITC_c).
+    underscore before the variable name (e.g. ``EITC_c`` vs ``_EITC_c``).
+
+    This class inherits methods from ParamTools like ``items``:
+
+        .. code-block :: python
+
+            import taxcalc as tc
+            pol = tc.Policy()
+
+            for name, value in pol.items():
+                print(name, value)
 
-    Note: Like all pt.Parameters classes the values of attributes
-    corresponding to a parameter value on this class are ephemeral and the only
-    way to make permanent changes to this class'sstate is through the set_state
-    or adjust methods.
+            # parameter_indexing_CPI_offset [0.]
+            # FICA_ss_trt [0.124]
+            # SS_Earnings_c [113700.]
+
+    Check out the ParamTools
+    `documentation <https://paramtools.dev/api/reference.html>`_
+    for more information on these inherited methods.
```

Comment:
Linking out to the ParamTools docs for inherited methods instead of showing them directly in the docs since the docstring formats are different and look a little wonky in the Tax-Calculator docs.